
{'date': 'Fri Dec 10 2021 13:29:33.421 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
RuntimeError: too much recursion
  module '$exec_2098' line 27
        self.vai()
'''},
{'date': 'Fri Dec 10 2021 13:29:53.946 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 31
    Gatil(GATIL_MOS).vai()
  module <module> line 28
    c0 = Elemento(img, x=140, y=340, w=200, h=200, cena=self)
NameError: name 'img' is not defined
'''},
{'date': 'Thu Feb 03 2022 18:45:31.150 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 45
    class Sala(SalaVit):
  module <module> line 46
    def __init__(self, n=NADA, l=NADA, s=NADA, o=NADA, nome='', **kwargs):
NameError: name 'NADA' is not defined
'''},
{'date': 'Fri Feb 04 2022 08:39:05.484 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 40
  class Hero(Elemento)
                       ^
SyntaxError: invalid syntax
'''},
{'date': 'Fri Feb 04 2022 08:40:01.86 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 47
  self.props ={P.H: Hero, P.T}
                              ^
SyntaxError: invalid syntax
'''},
{'date': 'Fri Feb 04 2022 08:40:39.545 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
Exception: bases[i].$dict is undefined
  module '$exec_339' line 5
class P(Enum):
'''},
{'date': 'Fri Feb 04 2022 09:40:04.520 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 48
  super().__init__(IMP.format(STRAY[0], x=x, y=y, w=w, h=h, cena=cena)
                                                                                ^
SyntaxError: invalid syntax
'''},
{'date': 'Fri Feb 04 2022 13:52:09.674 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 67
  self.nome = nome
            ^
SyntaxError: keyword can't be an expression
'''},
{'date': 'Fri Feb 04 2022 13:59:08.555 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 119
    Gatil(GATIL_MOS).vai()
  module <module> line 113
    et = Elemento(GITRAW, x=500, y=200, w=100, h=100, cena=sala_b.norte)
NameError: name 'GITRAW' is not defined
'''},
{'date': 'Fri Feb 04 2022 14:00:31.320 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 119
    Gatil(GATIL_MOS).vai()
  module <module> line 113
    et = Elemento(GITRAW, x=500, y=200, w=100, h=100, cena=sala_b.norte)
NameError: name 'GITRAW' is not defined
'''},
{'date': 'Fri Feb 04 2022 14:22:13.876 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 123
    Gatil(GATIL_MOS).vai()
  module <module> line 117
    document['g1111'].bind('click', self.et_vai)
KeyError: g1111
'''},
{'date': 'Fri Feb 04 2022 14:41:57.446 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 67
  s = svg.svg( width="300" height="300")
                            ^
SyntaxError: invalid syntax
'''},
{'date': 'Fri Feb 04 2022 14:49:51.339 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 128
    Gatil(GATIL_MOS).vai()
  module <module> line 125
    Thrash(sala_b.norte.elt)
  module <module> line 67
    s = svg.svg(xmlns="http://www.w3.org/2000/svg", version="1.1", viewBox="0 0 400 400", width="300", height="300")
ValueError: can't set attribute xmlns
'''},
{'date': 'Sat Feb 05 2022 11:46:52.649 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 147
    Gatil(GATIL_MOS).vai()
  module <module> line 145
    Thrash(x.elt, sala_b.norte)
  module <module> line 67
    cache = create_script_tag(GITRAW)
  module <module> line 87
    _tag = document.createElement('svg')
NameError: name 'document' is not defined
'''},
{'date': 'Sat Feb 05 2022 11:49:26.299 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
Exception: $B.get_class(...) is undefined
  module '$exec_320' line 148
Gatil(GATIL_MOS).vai()
'''},
{'date': 'Sat Feb 05 2022 11:51:58.772 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
Exception: $B.get_class(...) is undefined
  module '$exec_320' line 148
Gatil(GATIL_MOS).vai()
'''},
{'date': 'Mon Feb 07 2022 12:34:14.691 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 176
    Gatil(GATIL_MOS).vai()
  module <module> line 174
    Thrash(x.elt, sala_b.norte.elt)
  module <module> line 104
    s <= circle
NameError: name 'circle' is not defined
'''},
{'date': 'Mon Feb 07 2022 12:43:38.133 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 103
  obj = svg.use(href=f"#{label}" x= 50*dx, y=50*dy,
                                  ^
SyntaxError: invalid syntax
'''},
{'date': 'Mon Feb 07 2022 16:18:30.125 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 185
    Gatil(GATIL_MOS).vai()
  module <module> line 183
    Thrash(x.elt, sala_b.norte.elt)
  module <module> line 85
    cache = self.create_script_tag(LIXAO)
  module <module> line 122
    _fp = urllib.request.urlopen(src)
  module urllib.request line 37
    raise error. HTTPError('file not found')
HTTPError: file not found
'''},
{'date': 'Mon Feb 07 2022 17:57:51.66 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 187
    Gatil(GATIL_MOS).vai()
  module <module> line 185
    Thrash(x.elt, sala_b.norte.elt)
  module <module> line 111
    obj = svg.use(href=f"#{label}", x=5000 - 400*randint(0,9), y=300 -r200*randint(0,9), width=50, height=50,
NameError: name 'r200' is not defined
'''},
{'date': 'Wed Feb 09 2022 10:44:03.130 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 129
  def vai(self, ev):f"translate({dx} {dy})  rotate({7*indice} {ROFFX} {ROFFY}) scale(2.5)")
                      ^
SyntaxError: invalid syntax
'''},
{'date': 'Wed Feb 09 2022 18:13:23.270 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 139
  food = Elemento('', x=0, y=0, w=200, h=200 cena=self.cena)
                                              ^
SyntaxError: invalid syntax
'''},
{'date': 'Wed Feb 09 2022 19:21:59.982 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 200
    Gatil(GATIL_MOS).vai()
  module <module> line 198
    Thrash(x.elt, sala_b.norte)
  module <module> line 93
    elt <= s
NameError: name 's' is not defined
'''},
{'date': 'Wed Feb 09 2022 20:07:34.750 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 92
  def dump(self, cena, sorte=4)
                                ^
SyntaxError: invalid syntax
'''},
{'date': 'Wed Feb 09 2022 20:15:37.933 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 208
    Gatil(GATIL_MOS).vai()
  module <module> line 164
    self.trash = Thrash(x.elt, sala_b.norte)
AttributeError: 'int' object has no attribute 'elt'
'''},
{'date': 'Wed Feb 09 2022 20:17:26.282 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 209
    Gatil(GATIL_MOS).vai()
  module <module> line 178
    sala_a = Sala(*[IM.format(lnk) for lnk in SA])
  module <module> line 155
    self.cenas = [Rua(img) if isinstance(img, str) else img for img in [n, l, s, o]]
TypeError: __init__ missing 1 positional argument: 'trash'
'''},
{'date': 'Wed Feb 09 2022 20:34:39.752 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
Exception: $B.get_class(...) is undefined
  module '$exec_36683' line 208
Gatil(GATIL_MOS).vai()
'''},
{'date': 'Wed Feb 09 2022 20:53:07.955 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 92
  def dump(self, cena, sorte=4)
                                ^
SyntaxError: invalid syntax
'''},
{'date': 'Wed Feb 09 2022 21:01:47.10 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 208
    Gatil(GATIL_MOS).vai()
  module <module> line 182
    sala_b_args[0] = Rua(sala_b_args[0], self.trash, [
  module <module> line 78
    [p[proname](*proargs) for proname, proargs  in props]
  module <module> line 66
    super().__init__(HALO, x=x, y=y, w=w, h=h, o=0.2, vai=self.cena, cena=cena)
AttributeError: 'Trash' object has no attribute 'cena'
'''},
{'date': 'Wed Feb 09 2022 21:02:01.783 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 208
    Gatil(GATIL_MOS).vai()
  module <module> line 182
    sala_b_args[0] = Rua(sala_b_args[0], self.trash, [
  module <module> line 78
    [p[proname](*proargs) for proname, proargs  in props]
  module <module> line 66
    super().__init__(HALO, x=x, y=y, w=w, h=h, o=0.2, vai=self.cena, cena=cena)
AttributeError: 'Trash' object has no attribute 'cena'
'''},
{'date': 'Wed Feb 09 2022 21:47:17.12 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 2
    from _spy.vitollino.main import Cena, STYLE, Elemento, Sala as SalaVit, Labirinto, NADA, J, INVENTARIO as INV
ImportError: cannot import name 'J'

ImportError
Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 268
    action()
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 2
    from _spy.vitollino.main import Cena, STYLE, Elemento, Sala as SalaVit, Labirinto, NADA, J, INVENTARIO as INV
'''},
{'date': 'Wed Feb 09 2022 21:48:11.39 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 3
    from _spy.vitollino.main import J
ImportError: cannot import name 'J'

ImportError
Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 268
    action()
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 3
    from _spy.vitollino.main import J
'''},
{'date': 'Wed Feb 09 2022 21:48:38.933 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 293
    Gatil(GATIL_MOS).vai()
  module <module> line 291
    Swap(J, IM.format(CATPUZ),sala_b.norte)
  module <module> line 70
    class Peca(j.a):
AttributeError: 'Jogo' object has no attribute 'a'
'''},
{'date': 'Wed Feb 09 2022 21:51:46.343 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 293
    Gatil(GATIL_MOS).vai()
  module <module> line 291
    Swap(J, IM.format(CATPUZ),sala_b.norte)
  module <module> line 70
    class Peca(j.a):
AttributeError: 'Jogo' object has no attribute 'a'
'''},
{'date': 'Wed Feb 09 2022 21:53:04.994 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 293
    Gatil(GATIL_MOS).vai()
  module <module> line 291
    Swap(J(), IM.format(CATPUZ),sala_b.norte)
  module <module> line 122
    self.venceu = venceu or J.n(cena, "Voce venceu!")
AttributeError: 'Jogo' object has no attribute 'n'
'''},
{'date': 'Wed Feb 09 2022 22:40:53.511 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 99
  return compound([
                                                                                                 ^
SyntaxError: invalid syntax
'''},
{'date': 'Wed Feb 09 2022 22:42:26.3 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 99
  return compound([ellipsoid(size=(m, m, m / PHI), pos=d, color=cor),
                                                                                    ^
SyntaxError: invalid syntax
'''},
{'date': 'Wed Feb 09 2022 23:25:25.21 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 296
    Gatil(GATIL_MOS).vai()
  module <module> line 294
    Swap(J(), IM.format(CATPUZ),sala_b.norte,)
  module <module> line 122
    self.venceu = venceu or J.n(cena, "Voce venceu!")
AttributeError: 'Jogo' object has no attribute 'n'
'''},
{'date': 'Wed Feb 09 2022 23:26:23.961 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 296
    Gatil(GATIL_MOS).vai()
  module <module> line 294
    Swap(J(), IM.format(CATPUZ),sala_b.norte,)
  module <module> line 122
    self.venceu = venceu or J.n(cena, "Voce venceu!")
AttributeError: 'Jogo' object has no attribute 'n'
'''},
{'date': 'Wed Feb 09 2022 23:30:20.888 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 157
  self.puz = Swap(J(), IM.format(choice(CATPUZ)),cena, venceu=Cena(vai=self.foi)
                                                                                              ^
SyntaxError: invalid syntax
'''},
{'date': 'Wed Feb 09 2022 23:30:40.86 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 299
    Gatil(GATIL_MOS).vai()
  module <module> line 297
    Swap(J(), IM.format(CATPUZ),sala_b.norte,)
  module <module> line 122
    self.venceu = venceu or J.n(cena, "Voce venceu!")
AttributeError: 'Jogo' object has no attribute 'n'
'''},
{'date': 'Wed Feb 09 2022 23:31:50.643 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 299
    Gatil(GATIL_MOS).vai()
  module <module> line 297
    Swap(J(), IM.format(CATPUZ),sala_b.norte,)
  module <module> line 122
    self.venceu = venceu or J.n(cena, "Voce venceu!")
AttributeError: 'Jogo' object has no attribute 'n'
'''},
{'date': 'Thu Feb 10 2022 10:10:53.632 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 307
    Gatil(GATIL_MOS).vai()
  module <module> line 261
    self.trash = Thrash()
  module <module> line 183
    self.__vai = self.vai
AttributeError: 'Thrash' object has no attribute 'vai'
'''},
{'date': 'Thu Feb 10 2022 14:58:25.600 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 63
  posx, posy = -w//dx * (posicao % dx), -h//dy * (posicao // dx)
                                       ^
SyntaxError: invalid syntax
'''},
{'date': 'Thu Feb 10 2022 15:02:50.329 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 63
  posx, posy = -w//dx * (posicao % dx), -h//dy * (posicao // dx)
                                       ^
SyntaxError: invalid syntax
'''},
{'date': 'Thu Feb 10 2022 15:05:06.20 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 63
  posx, posy = -(w//dx) * (posicao % dx), -(h//dy) * (posicao // dx)
                                         ^
SyntaxError: invalid syntax
'''},
{'date': 'Thu Feb 10 2022 15:07:23.307 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 64
  posx, posy = -(w//dx) * (posicao % dx), -(h//dy) * (posicao // dx)
                                         ^
SyntaxError: invalid syntax
'''},
{'date': 'Thu Feb 10 2022 15:07:46.190 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 64
  posx, posy = -(w//dx) * (posicao % dx), -(h//dy) * (posicao // dx)
                                         ^
SyntaxError: invalid syntax
'''},
{'date': 'Thu Feb 10 2022 15:08:00.153 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 64
  posx, posy = -(w//dx) * (posicao % dx), -(h//dy) * (posicao // dx)
                                         ^
SyntaxError: invalid syntax
'''},
{'date': 'Thu Feb 10 2022 18:44:03.633 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 72
  self.linha.entra(esta_cena)esta_cena
                              ^
SyntaxError: invalid syntax
'''},
{'date': 'Thu Feb 10 2022 19:55:41.402 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 71
  sty = {'background-position': '{} {}'.format(posx, posy), 'background-size': '{} {}'.format(300, 150),
                                                                                                                                                                                                                                                                                                                                                                                           ^
SyntaxError: invalid syntax
'''},
{'date': 'Fri Feb 11 2022 10:30:09.521 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 388
    Gatil(GATIL_MOS).vai()
  module <module> line 342
    self.trash = Thrash()
  module <module> line 255
    hidden = Elemento['']
AttributeError: 'Elemento' object has no attribute '__getitem__'
'''},
{'date': 'Fri Feb 11 2022 10:30:41.698 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 388
    Gatil(GATIL_MOS).vai()
  module <module> line 342
    self.trash = Thrash()
  module <module> line 256
    hidden.setAttribute['hidden', 'hidden']
AttributeError: 'Elemento' object has no attribute 'setAttribute'
'''},
{'date': 'Fri Feb 11 2022 10:31:01.846 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 388
    Gatil(GATIL_MOS).vai()
  module <module> line 342
    self.trash = Thrash()
  module <module> line 256
    hidden.setAttribute('hidden', 'hidden')
AttributeError: 'Elemento' object has no attribute 'setAttribute'
'''},
{'date': 'Fri Feb 11 2022 10:32:51.463 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 389
    Gatil(GATIL_MOS).vai()
  module <module> line 343
    self.trash = Thrash()
  module <module> line 258
    pycard <= hidden
TypeError: can't add 'Elemento' object to DOMNode instance
'''},
{'date': 'Fri Feb 11 2022 11:50:14.184 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 362
  sala_b_args = [IM.format(lnk) for lnk in ESGOTO[1]]]
                                                     ^
SyntaxError: invalid syntax
'''},
{'date': 'Wed Mar 02 2022 12:26:09.72 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 413
    Gatil(GATIL_MOS).vai()
  module <module> line 408
    cur = Cursor(alvo=sala_b.norte)
  module _spy.vitollino.main line 1489
    dims = [self.alvo.style.top, self.alvo.style.minHeight, self.alvo.style.left, self.alvo.style.width]
AttributeError: 'Rua' object has no attribute 'style'
'''},
{'date': 'Wed Mar 02 2022 12:39:35.182 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 413
    Gatil(GATIL_MOS).vai()
  module <module> line 408
    cur = Cursor(alvo=g)
  module _spy.vitollino.main line 1489
    dims = [self.alvo.style.top, self.alvo.style.minHeight, self.alvo.style.left, self.alvo.style.width]
AttributeError: 'dict' object has no attribute 'top'
'''},
{'date': 'Wed Mar 02 2022 12:43:55.479 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 414
    Gatil(GATIL_MOS).vai()
  module <module> line 409
    cur = Cursor(alvo=c)
  module _spy.vitollino.main line 1489
    dims = [self.alvo.style.top, self.alvo.style.minHeight, self.alvo.style.left, self.alvo.style.width]
AttributeError: 'dict' object has no attribute 'top'
'''},
{'date': 'Wed Mar 02 2022 12:49:14.945 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 415
    Gatil(GATIL_MOS).vai()
  module <module> line 410
    cur = Cursor(alvo=c)
  module _spy.vitollino.main line 1489
    dims = [self.alvo.style.top, self.alvo.style.minHeight, self.alvo.style.left, self.alvo.style.width]
AttributeError: 'dict' object has no attribute 'top'
'''},
{'date': 'Wed Mar 02 2022 13:25:43.530 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module gatil.util line 17
  class Cursor(Elemento:
                        ^
SyntaxError: invalid syntax
'''},
{'date': 'Wed Mar 02 2022 13:26:24.339 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 416
    Gatil(GATIL_MOS).vai()
  module <module> line 411
    cur = Cursor(alvo=c)
TypeError: __init__ missing 1 positional argument: 'img'
'''},
{'date': 'Wed Mar 02 2022 13:28:07.182 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 414
    Gatil(GATIL_MOS).vai()
  module <module> line 409
    cur = Cursor(GATAR, x=200, y=250, w=100, h=100)
  module gatil.util line 128
    style.update(**_PATTERN.STARRY)
NameError: name 'style' is not defined
'''},
{'date': 'Wed Mar 02 2022 13:30:33.726 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 414
    Gatil(GATIL_MOS).vai()
  module <module> line 409
    cur = Cursor(GATAR, x=200, y=250, w=100, h=100)
  module gatil.util line 129
    style.update(**_PATTERN.STARRY)
AttributeError: 'CSSProperty' object has no attribute 'update'
'''},
{'date': 'Wed Mar 02 2022 13:44:49.942 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 145
    c = Cena("https://i.imgur.com/cQogon6.jpg").vai()
NameError: name 'Cena' is not defined
'''},
{'date': 'Wed Mar 02 2022 13:45:51.980 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 147
    m = Cursor("https://i.imgur.com/bo5bxUQ.jpg", x=200, y=250, w=100, h=100, cena=c)
  module <module> line 120
    self.cursor = self.noop
AttributeError: 'Cursor' object has no attribute 'noop'
'''},
{'date': 'Wed Mar 02 2022 13:47:19.711 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 149
    m = Cursor("https://i.imgur.com/bo5bxUQ.jpg", x=200, y=250, w=100, h=100, cena=c)
  module <module> line 120
    self.cursor = self.noop
AttributeError: 'Cursor' object has no attribute 'noop'
'''},
{'date': 'Wed Mar 02 2022 13:49:00.32 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''dim left, top =  [250, 100, 200, 100]
Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 147
    m = Cursor("https://i.imgur.com/bo5bxUQ.jpg", x=200, y=250, w=100, h=100, cena=c)
  module <module> line 133
    self.elt = html.DIV(Id="__cursor__", style=style, title="")
NameError: name 'html' is not defined
'''},
{'date': 'Wed Mar 02 2022 13:50:16.587 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''dim left, top =  [250, 100, 200, 100]

Exception: <TypeError: $B.get_class(...) is undefined>
'''},
{'date': 'Wed Mar 02 2022 13:53:51.335 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
Exception: <TypeError: $B.get_class(...) is undefined>
'''},
{'date': 'Wed Mar 02 2022 13:57:49.299 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''dim left, top =  [250, 100, 200, 100]
elt =  __cursor__

Exception: <TypeError: $B.get_class(...) is undefined>
'''},
{'date': 'Wed Mar 02 2022 14:01:03.854 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''dim left, top =  [250, 100, 200, 100]

Exception: <TypeError: $B.get_class(...) is undefined>
'''},
{'date': 'Wed Mar 02 2022 17:59:29.65 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 234
  eat = (time * (len(self.cats)+1)
                                            ^
SyntaxError: invalid syntax
'''},
{'date': 'Wed Mar 02 2022 18:00:09.243 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 433
    Gatil(GATIL_MOS).vai()
  module <module> line 404
    sala_b_args[0] = Rua(sala_b_args[0], self.trash, [
  module <module> line 273
    [p[proname](*proargs) for proname, proargs  in props]
TypeError: __init__() takes 1 positional argument but more were given
'''},
{'date': 'Wed Mar 02 2022 18:04:03.787 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 433
    Gatil(GATIL_MOS).vai()
  module <module> line 404
    sala_b_args[0] = Rua(sala_b_args[0], self.trash, [
  module <module> line 273
    [p[proname](*proargs) for proname, proargs  in props]
  module <module> line 245
    super().__init__(img=PETUNIO, x=x, y=y, w=w, h=h, cena=cena)
TypeError: __init__() got an unexpected keyword argument 'img'
'''},
{'date': 'Wed Mar 02 2022 18:09:28.821 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 433
    Gatil(GATIL_MOS).vai()
  module <module> line 404
    sala_b_args[0] = Rua(sala_b_args[0], self.trash, [
  module <module> line 273
    [p[proname](*proargs) for proname, proargs  in props]
  module <module> line 245
    super().__init__(x=x, y=y, w=w, h=h, cena=cena)
TypeError: __init__() got an unexpected keyword argument 'x'
'''},
{'date': 'Wed Mar 02 2022 18:25:17.427 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 453
    Gatil(GATIL_MOS).vai()
  module <module> line 424
    sala_b_args[0] = Rua(sala_b_args[0], self.trash, [
  module <module> line 293
    [p[proname](*proargs) for proname, proargs  in props]
  module <module> line 265
    self.enter(cena)
AttributeError: 'Hero' object has no attribute 'enter'
'''},
{'date': 'Wed Mar 02 2022 21:15:40.899 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
RuntimeError: too much recursion
  module '$exec_8562' line 481
Gatil(GATIL_MOS).vai()
'''},
{'date': 'Wed Mar 02 2022 21:17:18.363 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 454
    Gatil(GATIL_MOS).vai()
  module <module> line 425
    sala_b_args[0] = Rua(sala_b_args[0], self.trash, [
  module <module> line 294
    [p[proname](*proargs) for proname, proargs  in props]
  module <module> line 266
    self.enter(cena)
AttributeError: 'Hero' object has no attribute 'enter'
'''},
{'date': 'Wed Mar 02 2022 22:22:44.868 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
Exception: obj is undefined
  module '$exec_2701' line 18
class Cursor(Elemento):
'''},
{'date': 'Wed Mar 02 2022 22:25:33.689 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 127
  def _mouse_move(ev)u: return self.cursor.mouse_move(ev)
                      ^
SyntaxError: invalid syntax
'''},
{'date': 'Wed Mar 02 2022 22:25:49.808 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 178
    m = Cursor("https://i.imgur.com/bo5bxUQ.jpg", x=200, y=250, w=100, h=100, cena=c)
  module <module> line 23
    outer = selfu
NameError: name 'selfu' is not defined
'''},
{'date': 'Wed Mar 02 2022 22:27:01.580 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 178
    m = Cursor("https://i.imgur.com/bo5bxUQ.jpg", x=200, y=250, w=100, h=100, cena=c)
  module <module> line 23
    outer = selfu
NameError: name 'selfu' is not defined
'''},
{'date': 'Wed Mar 02 2022 22:27:51.135 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
Exception: <TypeError: obj is undefined>
'''},
{'date': 'Wed Mar 02 2022 22:30:34.843 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
Exception: <TypeError: obj is undefined>
'''},
{'date': 'Wed Mar 02 2022 22:32:23.24 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
Exception: <TypeError: obj is undefined>
'''},
{'date': 'Wed Mar 02 2022 22:33:20.185 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
Exception: <TypeError: obj is undefined>
'''},
{'date': 'Wed Mar 02 2022 22:34:10.931 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
Exception: <TypeError: obj is undefined>
'''},
{'date': 'Wed Mar 02 2022 23:00:13.400 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 447
    Gatil(GATIL_MOS).vai()
  module <module> line 441
    sala_b.norte.vai()
  module <module> line 293
    TheHero().turn()
  module <module> line 236
    GATIL.turn()
AttributeError: 'NoneType' object has no attribute 'turn'
'''},
{'date': 'Wed Mar 02 2022 23:04:52.181 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 447
    Gatil(GATIL_MOS).vai()
  module <module> line 441
    sala_b.norte.vai()
  module <module> line 293
    TheHero().turn()
  module <module> line 239
    eat = time * (len(self.cats)+1)
AttributeError: 'TheHero' object has no attribute 'cats'
'''},
{'date': 'Wed Mar 02 2022 23:07:11.559 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 447
    Gatil(GATIL_MOS).vai()
  module <module> line 441
    sala_b.norte.vai()
  module <module> line 293
    TheHero().turn()
  module <module> line 236
    GATIL.turn()
AttributeError: 'NoneType' object has no attribute 'turn'
'''},
{'date': 'Fri Mar 04 2022 12:19:54.600 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 236
  setattr(TheHero, key[2:], property(lambda *_: TheHero.PROFILE[key], self.fn_readyonly)
                                                                                               ^
SyntaxError: invalid syntax
'''},
{'date': 'Fri Mar 04 2022 12:21:26.397 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 238
  def fn_readonly(self, v)
                           ^
SyntaxError: invalid syntax
'''},
{'date': 'Fri Mar 04 2022 12:22:18.192 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 461
    Gatil(GATIL_MOS).vai()
  module <module> line 436
    sala_a_args = [Rua(sala, self.trash,[(P.H, [200, 550])]) for sala in sala_a_args]
  module <module> line 297
    [p[proname](*proargs) for proname, proargs  in props]
  module <module> line 267
    self.hero = TheHero()
  module <module> line 225
    self.start()
  module <module> line 236
    setattr(TheHero, key[2:], property(lambda *_: TheHero.PROFILE[key], self.fn_readyonly))
AttributeError: 'TheHero' object has no attribute 'fn_readyonly'
'''},
{'date': 'Fri Mar 04 2022 12:23:28.988 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 236
  setattr(TheHero, key[2:], property(lambda *_: TheHero.PROFILE[key], lambda *_:  raise "It is ready only"))
                                                                                   ^
SyntaxError: invalid syntax
'''},
{'date': 'Fri Mar 04 2022 12:23:53.131 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 461
    Gatil(GATIL_MOS).vai()
  module <module> line 455
    sala_b.norte.vai()
  module <module> line 305
    TheHero().turn()
TypeError: 'int' object is not callable
'''},
{'date': 'Fri Mar 04 2022 13:29:31.859 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 314
  hidden = Elemento('', style="position:absolute; top:-2000; left:-2000;)
                                                                         ^
SyntaxError: EOL while scanning string literal
'''},
{'date': 'Fri Mar 04 2022 20:08:35.502 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
Exception: <TypeError: $range.tree[0] is undefined>
'''},
{'date': 'Fri Mar 04 2022 20:09:01.810 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 470
    Gatil(GATIL_MOS).vai()
  module <module> line 464
    sala_b.norte.vai()
  module <module> line 309
    TheHero().do_turn()
  module <module> line 256
    eaten = self.cats + 1
TypeError: can only concatenate list (not "int") to list
'''},
{'date': 'Fri Mar 04 2022 20:55:33.887 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 222
  class TheHero()Cursor) #Elemento):
                  ^
SyntaxError: invalid syntax
'''},
{'date': 'Fri Mar 04 2022 20:55:43.982 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 222
  class TheHero(Cursor) #Elemento):
                                    ^
SyntaxError: invalid syntax
'''},
{'date': 'Fri Mar 04 2022 21:02:06.650 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 476
    Gatil(GATIL_MOS).vai()
  module <module> line 470
    sala_b.norte.vai()
  module <module> line 314
    TheHero().do_turn()
  module <module> line 262
    [INV.tira(fishes.pop()) for _ in range(eaten)] if len(fishes) >= eaten else self.game_over()
  module _spy.vitollino.main line 383
    item_img = document[nome_item]
KeyError: 13_fish
'''},
{'date': 'Fri Mar 04 2022 21:03:32.179 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 476
    Gatil(GATIL_MOS).vai()
  module <module> line 470
    sala_b.norte.vai()
  module <module> line 314
    TheHero().do_turn()
  module <module> line 262
    [INV.tira(fishes.pop()) for _ in range(eaten)] if len(fishes) >= eaten else self.game_over()
  module _spy.vitollino.main line 383
    item_img = document[nome_item]
KeyError: 8_fish
'''},
{'date': 'Fri Mar 04 2022 21:05:49.379 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 476
    Gatil(GATIL_MOS).vai()
  module <module> line 470
    sala_b.norte.vai()
  module <module> line 314
    TheHero().do_turn()
  module <module> line 262
    [INV.tira(fishes.pop()) for _ in range(eaten)] if len(fishes) >= eaten else self.game_over()
  module _spy.vitollino.main line 383
    item_img = document[nome_item]
KeyError: 3_fish
'''},
{'date': 'Fri Mar 04 2022 21:06:51.427 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 476
    Gatil(GATIL_MOS).vai()
  module <module> line 443
    self.gatar = g = Elemento(GATAR, x=200, y=550, w=100, h=100, vai=h.resposta)
AttributeError: 'TheHero' object has no attribute 'resposta'
'''},
{'date': 'Fri Mar 04 2022 21:07:58.802 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 477
    Gatil(GATIL_MOS).vai()
  module <module> line 444
    self.gatar = g = Elemento(GATAR, x=200, y=550, w=100, h=100, vai=h.resposta)
AttributeError: 'TheHero' object has no attribute 'resposta'
'''},
{'date': 'Fri Mar 04 2022 21:10:07.574 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 1
  # ada.gatil.main.py
                               ^
SyntaxError: invalid syntax
'''},
{'date': 'Fri Mar 04 2022 21:11:38.632 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 478
    Gatil(GATIL_MOS).vai()
  module <module> line 472
    sala_b.norte.vai()
  module <module> line 315
    TheHero().do_turn()
  module <module> line 263
    [INV.tira(fishes.pop()) for _ in range(eaten)] if len(fishes) >= eaten else self.game_over()
  module _spy.vitollino.main line 383
    item_img = document[nome_item]
KeyError: 13_fish
'''},
{'date': 'Fri Mar 04 2022 21:14:50.152 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 478
    Gatil(GATIL_MOS).vai()
  module <module> line 472
    sala_b.norte.vai()
  module <module> line 315
    TheHero().do_turn()
  module <module> line 263
    [INV.tira(fishes.pop()) for _ in range(eaten)] if len(fishes) >= eaten else self.game_over()
  module _spy.vitollino.main line 383
    item_img = document[nome_item]
KeyError: 3_fish
'''},
{'date': 'Fri Mar 04 2022 21:20:45.229 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 479
    Gatil(GATIL_MOS).vai()
  module <module> line 454
    sala_a_args = [Rua(sala, self.trash,[(P.H, [200, 550])]) for sala in sala_a_args]
  module <module> line 308
    [p[proname](*proargs) for proname, proargs  in props]
  module <module> line 277
    self.hero = TheHero()
  module <module> line 228
    self.start()
  module <module> line 232
    GATIL.gatar.vai = self.limpa
AttributeError: 'NoneType' object has no attribute 'gatar'
'''},
{'date': 'Fri Mar 04 2022 21:24:19.403 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 479
    Gatil(GATIL_MOS).vai()
  module <module> line 474
    g.vai = TheHero().resultado
AttributeError: 'TheHero' object has no attribute 'resultado'
'''},
{'date': 'Fri Mar 04 2022 21:36:40.834 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 483
    Gatil(GATIL_MOS).vai()
  module <module> line 477
    g.vai = h.resposta
AttributeError: 'TheHero' object has no attribute 'resposta'
'''},
{'date': 'Fri Mar 04 2022 21:39:05.677 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 481
    Gatil(GATIL_MOS).vai()
  module <module> line 476
    g.vai = TheHero().resultado
AttributeError: 'TheHero' object has no attribute 'resultado'
'''},
{'date': 'Fri Mar 04 2022 21:41:21.559 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 483
    Gatil(GATIL_MOS).vai()
  module <module> line 476
    h = TheHero().cursor
AttributeError: 'TheHero' object has no attribute 'cursor'
'''},
{'date': 'Fri Mar 04 2022 21:44:59.603 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 481
    Gatil(GATIL_MOS).vai()
  module <module> line 455
    sala_a_args = [Rua(sala, self.trash,[(P.H, [200, 550])]) for sala in sala_a_args]
  module <module> line 309
    [p[proname](*proargs) for proname, proargs  in props]
  module <module> line 278
    self.hero = TheHero()
  module <module> line 227
    self.start()
  module <module> line 232
    GATIL.g.vai = resposta
AttributeError: 'NoneType' object has no attribute 'g'
'''},
{'date': 'Fri Mar 04 2022 21:49:29.909 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 480
    Gatil(GATIL_MOS).vai()
  module <module> line 455
    sala_a_args = [Rua(sala, self.trash,[(P.H, [200, 550])]) for sala in sala_a_args]
  module <module> line 309
    [p[proname](*proargs) for proname, proargs  in props]
  module <module> line 278
    self.hero = TheHero()
  module <module> line 227
    self.start()
  module <module> line 232
    INV.item["gatar"].vai = resposta
NameError: name 'resposta' is not defined
'''},
{'date': 'Mon Mar 28 2022 13:00:44.315 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 510
  TODOS = [{PERGUTA: "Qual do nomes abaixo é um vídeo da Flávia?", 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         ^
SyntaxError: invalid syntax
'''},
{'date': 'Mon Mar 28 2022 13:01:01.88 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 510
  TODOS = [{PERGUNTA: "Qual do nomes abaixo é um vídeo da Flávia?", 
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          ^
SyntaxError: invalid syntax
'''},
{'date': 'Mon Mar 28 2022 13:01:48.897 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 7
    from random import randint, shuffle, choice, choices
ImportError: cannot import name 'choices'

ImportError
Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 268
    action()
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 7
    from random import randint, shuffle, choice, choices
'''},
{'date': 'Mon Mar 28 2022 16:45:13.535 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 224
    class TheHero(Elemento):
  module <module> line 233
    def start(self, cena=cena):
NameError: name 'cena' is not defined
'''},
{'date': 'Mon Apr 04 2022 13:42:08.195 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 42
    SALAS = [S1, S2, S3, S4, S5, S6, S7, S8, S9]
NameError: name 'S4' is not defined
'''},
{'date': 'Mon Apr 04 2022 13:56:35.239 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 582
    Gatil(GATIL_MOS).vai()
  module <module> line 531
    NV.bota(p)
NameError: name 'NV' is not defined
'''},
{'date': 'Mon Apr 04 2022 14:03:10.498 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 583
    Gatil(GATIL_MOS).vai()
  module <module> line 537
    alert(sala_args)
NameError: name 'sala_args' is not defined
'''},
{'date': 'Mon Apr 04 2022 14:09:06.2 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 583
    Gatil(GATIL_MOS).vai()
  module <module> line 548
    sala_b_args =  SI5 #[IM.format(lnk) for lnk in SB]
NameError: name 'SI5' is not defined
'''},
{'date': 'Mon Apr 04 2022 17:23:05.753 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 596
    Gatil(GATIL_MOS).vai()
  module <module> line 546
    sala_a_args[0] = Rua(sala_a_img[0], self.trash, [(P.P, [f"UMA RUA {0}"])
TypeError: 'tuple' object is not callable
'''},
{'date': 'Mon Apr 04 2022 17:23:17.148 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 596
    Gatil(GATIL_MOS).vai()
  module <module> line 546
    sala_a_args[0] = Rua(sala_a_img[0], self.trash, [(P.P, [f"UMA RUA {0}"]),
  module <module> line 391
    [p[proname](*proargs) for proname, proargs  in props]
  module <module> line 305
    super().__init__(HALO, x=x, y=y, w=w, h=h, cena=cena)
TypeError: object() takes no parameters
'''},
{'date': 'Mon Apr 04 2022 18:03:28.943 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 601
    Gatil(GATIL_MOS).vai()
  module <module> line 544
    salas_args = [[Rua(sala, self.trash,[(P.H, [200, 550]), (P.P, [f"RUA {R[rua]} {C[canto]}"])]), (P.S, [1043, 473])
TypeError: append() takes 2 positional argument but more were given
'''},
{'date': 'Mon Apr 04 2022 18:22:06.446 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 603
    Gatil(GATIL_MOS).vai()
  module <module> line 551
    sala_a_args[0] = Rua(sala_a_img[0], self.trash, [(P.P, [f"RUA MENDES {C[0]}"]),
IndexError: pop index out of range
'''},
{'date': 'Wed Apr 06 2022 10:53:33.3 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 607
    Gatil(GATIL_MOS).vai()
  module <module> line 549
    salas_args = [[Rua(sala, self.trash,[(P.H, [200, 550]), (P.S, [1043, (573+G.pop()) if rua in Z else 573]),
TypeError: append() takes 2 positional argument but more were given
'''},
{'date': 'Wed Apr 06 2022 10:53:45.275 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 607
    Gatil(GATIL_MOS).vai()
  module <module> line 549
    salas_args = [[Rua(sala, self.trash,[(P.H, [200, 550]), (P.S, [1043, (573+G.pop()) if rua in Z else 573]),
TypeError: append() takes 2 positional argument but more were given
'''},
{'date': 'Wed Apr 06 2022 11:08:25.328 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 529
  return pos+collection.pop()) if R[rua] in Z else pos
                             ^
SyntaxError: invalid syntax
'''},
{'date': 'Wed Apr 06 2022 11:08:56.590 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
NameError: cannot assign to function call
  module '$exec_4450' line 609
Gatil(GATIL_MOS).vai()
'''},
{'date': 'Wed Apr 06 2022 12:46:02.369 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 630
    Gatil(GATIL_MOS).vai()
  module <module> line 546
    shuffle(Y)
NameError: name 'Y' is not defined
'''},
{'date': 'Wed Apr 06 2022 17:50:13.687 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 282
  texto = (f"Total de pontos: {self.xper}. Você alcançou o nível {self.levl} em {self.turn} turnos."+
                                                                                                                                                                                                          ^
SyntaxError: invalid syntax
'''},
{'date': 'Wed Apr 06 2022 18:06:32.940 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 644
    Gatil(GATIL_MOS).vai()
  module <module> line 610
    sala_b.norte.vai()
  module <module> line 413
    TheHero().do_turn()
  module <module> line 295
    self.learn(self.lvl+2)
AttributeError: 'TheHero' object has no attribute 'lvl'
'''},
{'date': 'Wed Apr 06 2022 18:08:02.700 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 644
    Gatil(GATIL_MOS).vai()
  module <module> line 610
    sala_b.norte.vai()
  module <module> line 413
    TheHero().do_turn()
  module <module> line 295
    self.learn(self.lvl+2)
AttributeError: 'TheHero' object has no attribute 'lvl'
'''},